pkg/controllers/user/logging/clusterlogging.go

```diff
@@ -298,7 +298,12 @@ func provision(namespaces v1.NamespaceInterface, configmaps v1.ConfigMapInterfac
 		return err
 	}
 
-	return utils.CreateFluentd(daemonsets, serviceAccounts, clusterRoleBindings, loggingconfig.LoggingNamespace, dockerRootDir)
+	cluster, err := clusterLister.Get("", clusterName)
```

Comment:
should we handle not found error specially? for the case when cluster is being removed.

Reply:
if the cluster is removed, these controllers will also be removed finally, maybe we don't need to handle the not found error?